lib/JSONAbstract.php

```diff
@@ -1,19 +1,64 @@
 <?php
 
-/**
- * An alternative abstract class for bridges utilizing JSON input
- *
- * This class is meant as an alternative base class for bridge implementations.
- * It offers preliminary functionality for generating feeds based on XPath
- * expressions.
- * As a minimum, extending classes should define XPath expressions pointing
- * to the feed items contents in the class constants below. In case there is
- * more manual fine tuning required, it offers a bunch of methods which can
- * be overridden, for example in order to specify formatting of field values
- * or more flexible definition of dynamic XPath expressions.
- *
- * This class extends {@see BridgeAbstract}, which means it incorporates and
- * extends all of its functionality.
- **/
-
-abstract class XPathAbstract extends BridgeAbstract {
+abstract class JSONAbstract extends UserDefinedAbstract {
+
+	const USER_EXPRESSION_DELIMITER = '.';
+
+	protected function provideWebsiteContent() {
+		$header = array(
+			'Content-Type: application/json'
```

Comment:
I think you want the accept header here. https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept

Reply:
Fixed